rocky/rocky/views/upload_raw.py

```diff
@@ -70,3 +80,25 @@ def process_raw(self, form):
             return self.add_error_notification(_("Raw file could not be uploaded to Bytes: %s") % str(exc))
         else:
             self.add_success_notification(_("Raw file successfully added."))
+
+    def get_form_kwargs(self):
+        kwargs = {
+            "connector": self.octopoes_api_connector,
+            "ooi_list": self.get_ooi_options(),
+        }
+        kwargs.update(super().get_form_kwargs())
+
+        if "ooi_class" in kwargs:
+            del kwargs["ooi_class"]
+
+        return kwargs
+
+    def get_ooi_options(self) -> list[tuple[str, str]]:
+        objects = self.octopoes_api_connector.list_objects(
+            set(OOI_TYPES.values()), valid_time=datetime.now(timezone.utc)
+        ).items
```

Comment:
Wont this return the *entire* list of OOI's? Rendering this form widget unusable for large databases, and possibly making the entire page unloadable?
While I do like the user to have some clicking options, maybe we should opt for a datalist (in which the user Can type an OOI-primary-key themself), or select from the 100 most recently added ooi's or something.
Ideally, we'd move to a type-ahead style search-box, instead of a complete lit, or limited list.
This same problem is present on other create-ooi relation form fields.

Reply:
`list_objects()` has a default limit of 20, so this should not be a problem

Reply:
But with only 20, how can the user select the correct OOI if its beyond that?

Reply:
I believe the default limit is 50*